libs/components/forms/src/lib/modules/input-box/input-box-control.directive.ts

```diff
@@ -9,4 +12,20 @@ import { Directive } from '@angular/core';
     'input:not([skyId]):not(.sky-form-control),select:not([skyId]):not(.sky-form-control),textarea:not([skyId]):not(.sky-form-control)',
   standalone: true,
 })
-export class SkyInputBoxControlDirective {}
+export class SkyInputBoxControlDirective {
+  @HostBinding('autocomplete')
+  @Input()
+  public set autocomplete(value: string | undefined) {
+    this.#_autocomplete = value;
+    if (!value && this.#hostService) {
+      this.#_autocomplete = 'off';
+    }
+  }
+
+  public get autocomplete(): string | undefined {
+    return this.#_autocomplete;
+  }
+
+  #_autocomplete: string | undefined;
+  #hostService = inject(SkyInputBoxHostService, { optional: true });
```

Comment:
Did we ever settle on `Svc` vs. `Service` for naming service properties?

Reply:
The code conventions wiki does use `Svc` though I don't see it called out. I updated to `hostSvc`.